package.json

```diff
@@ -79,7 +79,7 @@
     "@kyvejs/sdk": "^1.3.2",
     "@solana/web3.js": "^1.91.7",
     "arbundles": "0.9.9",
-    "axios": "^1.4.0",
+    "axios": "1.4.0",
```

Comment:
seeing some type issues with axiosRetry when this is pulling in a latest version 🤔

Reply:
I'll take a lookin- we resolved this in ar-io-sdk